frontend/src/app/modules/availability/new-slot/questions/questions.component.sass

```diff
@@ -58,11 +81,30 @@ $question-input-width: 337px
     .list-item-wrapper .question-index
         cursor: ns-resize
 
+.drag-container
+    border: 2px solid $light-gray
+    border-radius: 10px
+    margin: 10px
+
+.button-container
+    display: flex
+    align-items: center
+    justify-content: center
```

Comment:
```suggestion
```

Just use `.flex` class. I believe `align-items` and `justify-content` are not needed here